Cython/Compiler/Symtab.py

```diff
@@ -1627,6 +1627,20 @@ def declare_var(self, name, type, pos,
             if not (type.is_pyobject and not type.is_extension_type):
                 raise InternalError(
                     "Non-cdef global variable is not a generic Python object")
+        if (is_cdef and visibility != "extern"
+                and self.directives['subinterpreters_compatible'] != "False"):
+            extra_warning = ""
+            pyobject_warning = ""
+            if type.is_pyobject:
+                extra_warning = ".\nPython objects should not be shared between interpreters"
+                pyobject_warning = "Python "
+            warning(
+                pos,
+                f"Global cdef {pyobject_warning}variable used with subinterpreter support enabled.\n"
+                "This variable is  not currently in the per-interpreter module state "
+                "but this will likely change in future releases" +
```

Comment:
We already have more than one sentence here, so no reason not to have a dot at the end of the second as well.
```suggestion
                "but this will likely change in future releases." +
```